app/config/index.js

```diff
@@ -0,0 +1,3 @@
+"use strict";
+
+module.exports.Config = require('./config');
```

Comment:
Editors like webstorm help a lot in these types of refactoring, but alright, might make sense for ES6. :) Been using Typescript for quite some time now, and if you try to do this type of refactoring, it fails to compile and you get the help you need to move things around. 

But I also see the point that the diff will be unnecessarily huge when you just want to move one of the files. No silver bullet here. But we prefer not re-exposing to avoid more boilerplate and potential errors.